roles/mesos-master/files/consul_template_config

```diff
@@ -0,0 +1,7 @@
+consul = "127.0.0.1:8500"
```

Comment:
I'm not 100% sure but I think Consul only responds to the private IP address, so if it's not working that might be why.